### DISCUSSION TITLE
virtual keyboard - autotype

### DISCUSSION THREAD

**Discussion by oldtigerin** [keepassxreboot/keepassxc#8559] [Q&A] [answered]
on github docs:
https://github.com/keepassxreboot/keepassxc/blob/develop/docs/topics/AutoType.adoc
droidmonkey explains that autotype acts like a virtual keyboard.
A virtual keyboard in windows is "immune" for keyloggers thus that autotype in keepassXC is unreachable for keyloggers?
I ask it because I have already seen on forums persons that pretends thata autotype in keepassXC is accessible by keyloggers.

**Comment by droidmonkey** [comment, marked as answer]
>  virtual keyboard in windows is "immune" for keyloggers

Where did you get this info? It is not correct.

**Comment by oldtigerin** [comment]
Hi,
I've read on the microsoft community aboout the windows virtual keyboard. Maybe this on-screen keyboard does not present all virtual keyboards or the person his answer is not correct.
https://answers.microsoft.com/en-us/windows/forum/all/does-the-windows-virtual-on-screen-keyboard-have/ea0c0f6e-edfc-4f8e-9b10-4cd086baa3d1

**Reply by droidmonkey** [reply]
First off that is from 2010, secondly the accepted answer is false. Expand the replies at the bottom....

**Comment by oldtigerin** [comment]
I have no experience about the level of the microsoft community in the usa but I trust your answer and knowledge so I will look for a good anti-keylogger software.
